database/models.py

```diff
@@ -71,3 +71,22 @@ class Snapshot(Base):
     trial = sqlalchemy.orm.relationship('Trial', back_populates='snapshots')
     edges_covered = Column(Integer, nullable=False)
     fuzzer_stats = Column(JSON, nullable=True)
+
+
+class Crash(Base):
+    """Represents crashes found in experiments."""
+    __tablename__ = 'crash'
+
+    experiment = Column(String,
+                        ForeignKey('experiment.name'),
+                        nullable=False,
+                        primary_key=True)
+    fuzzer = Column(String, nullable=False, primary_key=True)
+    benchmark = Column(String, nullable=False, primary_key=True)
+    crash_key = Column(String, nullable=False, primary_key=True)
+    crash_type = Column(String, nullable=False)
+    crash_address = Column(String, nullable=False)
+    crash_state = Column(String, nullable=False)
+    crash_stacktrace = Column(String, nullable=False)
+    crash_testcase = Column(String, nullable=False)
+    crash_time = Column(Integer, nullable=False)
```

Comment:
No trial?

Reply:
So, we are keeping deduplicated crashes here in this DB. Do we need trial id info, i can add it but is it that important ?